Documentation/event-reference.md

```diff
@@ -0,0 +1,204 @@
+
+| Value | Behavior |
+| ------ | -------- |
+| Yes (`y`) | Hits are sent |
+| No (`n`) | Hits are dropped and not sent |
+| Pending (`p`) | Hits are queued until `y`/`n` is set; when set, queued events follow the value's behavior |
+
+#### Event Details<!-- omit in toc -->
+
+| Event type                        | Event source                          |
+| --------------------------------- | ------------------------------------- |
+| com.adobe.eventType.edgeConsent   | com.adobe.eventSource.responseContent |
+
+#### Data payload definition<!-- omit in toc -->
+
+| Key       | Value type    | Required | Description           |
+| --------- | ------------- | -------- | --------------------- |
+| consents  | `Map<String, Object>` | No       | XDM formatted consent preferences containing current collect consent settings. If not specified, defaults to pending (`p`) until the value is updated. |
+
+----- 
+
+### Edge identity reset complete
+
+This event signals that [Identity for Edge Network](https://github.com/adobe/aepsdk-edgeidentity-android) has completed [resetting all identities](https://developer.adobe.com/client-sdks/documentation/identity-for-edge-network/api-reference/#resetidentities) usually following a call to [`MobileCore.resetIdentities()`](https://github.com/adobe/aepsdk-core-android/blob/main/Documentation/MobileCore/api-reference.md). 
+
+When this event is received, the Edge extension queues it up and removes the cached internal state:store settings. If other events are queued before this event, these events will be processed first in the order they were received.
+
+#### Event Details<!-- omit in toc -->
+
+| API                             | Event type                 | Event source                         |
+| ------------------------------- | -------------------------- | ------------------------------------ |
+| [`MobileCore.resetIdentities()`](https://github.com/adobe/aepsdk-core-android/blob/main/Documentation/MobileCore/api-reference.md) | com.adobe.eventType.edgeIdentity   | com.adobe.eventSource.resetComplete  |
+
+#### Data payload definition<!-- omit in toc -->
+
+This event has no data payload.
+
+-----
+
+## Events dispatched by Edge
+
+The following events are dispatched by the Edge extension client-side.
+
+### Edge identity response
+
+This event is a response to the [Edge request identity event](#edge-request-identity)) and provides the location hint being used by the Edge Network extension in requests to the Edge Network. The Edge Network location hint may be used when building the URL for Edge Network requests to hint at the server cluster to use.
+
+| Value | Behavior |
```

Comment:
We dont need this table here.

Reply:
We should add this is dispatched for edge-request-identity event with data containing` lochint = true`

Reply:
Also add a local href to the section getLocHint